### DISCUSSION TITLE
Sunshine host with PipeWire only sends microphone input to client

### DISCUSSION THREAD

**Discussion by nixRidge** [LizardByte/community#11] [Sunshine] [answered]
Hello! I'm not sure what info exactly would be helpful, but I'll try to clarify as much as I can.
Not sure if this deserves an issue or if I'm wrong to even open this post, so I'm taking my chances and asking here first.

I'd like Sunshine to use my desktop's audio output for audio loopback, but I can only hear my microphone on the client (Moonlight on Android).

I've built Sunshine/nightly and running it on my Arch Linux desktop, using KDE Plasma 5.24 X11 and PipeWire 0.3.45 exclusively, without any native PulseAudio or JACK packages installed.
I understand that there are some issues with Sunshine and PipeWire, and was hoping someone has found a workaround for this.

In my Sunshine configuration, I've set my audio sink to "alsa_output.pci-0000_29_00.1.hdmi-stereo-extra3" which is the sound output of my Nvidia GPU, via HDMI. I also have a USB mixer I use for my headphones, but setting the sink to that output didn't work either.

Just in case, here's my sunshine.conf
[sunshine.conf.txt](https://github.com/SunshineStream/Sunshine/files/8056954/sunshine.conf.txt)

Here's also a very verbose inxi output.
[inxi_fridgeracer.txt](https://github.com/SunshineStream/Sunshine/files/8056941/inxi_fridgeracer.txt)

I have not tried with PulseAudio installed, as I would like to find a way to keep running PipeWire only, but if it would help in any way I can install and test with PulseAudio if requested.

Let me know if there's anything else I can add that might help. Thanks in advance!

**Comment by pogojotz** [comment, marked as answer]
Hi @FridgeRacer.
This worked for me:
- Leave the Audio Sink in the Sunshine configuration empty
- Connect a moonlight client
- Start pavucontrol 
  - Switch to "Recording" tab
  - Select "Monitor of sink-sunshine-stereo" as recording source for sunshine

That was it... The setting is persistent for me across restarts.

**Reply by nixRidge** [reply]
Hey @pogojotz, thank you so much and my apologies for being so late to get back to you. I finally had a chance to test this out and it works like a charm.

You have my gratitude.
